components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -196,6 +196,7 @@ be lost if SCREAM_HACK_XML is not enabled.
       <do_predict_nc>true</do_predict_nc>
       <do_predict_nc COMPSET=".*SCREAM.*noAero">false</do_predict_nc>
       <enable_column_conservation_checks>false</enable_column_conservation_checks>
+      <max_total_ni type="real" doc="maximum total ice concentration (sum of all categories) (m)">740.0e3</max_total_ni>
```

Comment:
What's the trailing `(m)`?

Reply:
Are there constraints on this param? I'm guessing it should be non negative? If so, you could add `constraints="ge 0"` (or gt if it should be strictly positive). This would prevent an accidental typo like `atmchange max_total_ni=-800e3` from going through (the `=` and `-` keys are close together, so it's not unlikely to accidentally type =- instead of =).